Accompany/Views/TodoCell.swift

```diff
@@ -37,15 +44,19 @@ class TodoCell: UITableViewCell {
       make.left.equalTo(contentView).offset(10)
       make.right.equalTo(contentView).offset(10)
```

Comment:
Please remove line 44 and 45 as well.
```suggestion
      make.edges.equalTo(contentView).inset(UIEdgeInsets(top: 0, left: 10, bottom: 0, right: 10))
```